apps/judicial-system/web/messages/Core/index.ts

```diff
@@ -273,6 +273,11 @@ export const core = defineMessages({
     defaultMessage: 'Loka glugga',
     description: 'Notaður fyrir texta í Loka glugga takka í modölum.',
   },
+  stopModal: {
```

Comment:
Please consider moving this string to the local string file for `DenyIndictmentCaseModal`.